server/src/main/java/io/crate/replication/logical/LogicalReplicationService.java

```diff
@@ -218,6 +225,7 @@ private Runnable withRetry(Runnable runnable) {
     @Override
     public void close() throws IOException {
         metadataTracker.close();
+        shardReplicationService.close();
```

Comment:
This is not related to STOP SUBSCRIPTION, rather follow up to https://github.com/crate/crate/commit/01d8da73d38d7c4507d4b0b7fb9c86b2e45d9096

could be 
```
toClose.add(() -> stopWatch.stop().start("shard_replication_service"));
toClose.add(injector.getInstance(ShardReplicationService.class));
```

but I did it via `LogicalReplicationService.close`

Reply:
UPD:  Probably don't need it since we close trackers on Shard close/removal https://github.com/crate/crate/blob/logical-replication/server/src/main/java/io/crate/replication/logical/ShardReplicationService.java#L154